source/docs/casper/resources/tutorials/beginner/cep78/index.md

```diff
@@ -0,0 +1,251 @@
+- [_metadata_mutability_](https://github.com/casper-ecosystem/cep-78-enhanced-nft#metadata-mutability): The MetadataMutability modality dictates whether the metadata of minted NFTs can be updated. This argument is passed in as a u8 value and is required at the time of installation.
+
+### Example installation with Session Arguments {#session-arguments-example}
+For this tutorial, please install an instance of Cep78 using the following command and replace the [JSON SCHEMA ESCAPED STRING] with the example ESCAPED STRING that's explained in-depth below the command.
+```bash
+casper-client put-deploy \
+  --session-path ./contract/target/wasm32-unknown-unknown/release/contract.wasm \
+  --chain-name casper-test \
+  --node-address http://195.201.167.179:7777 \
+  --secret-key ./secret_key.pem \
+  --payment-amount 300000000000 \
+  --session-arg "collection_name:string='MY_COLLECTION'" \
+  --session-arg "collection_symbol:string='MY_SYMBOL'" \
+  --session-arg "total_token_supply:u64='1000'" \
+  --session-arg "ownership_mode:u8='2'" \
+  --session-arg "nft_kind:u8='2'" \
+  --session-arg "identifier_mode:u8='1'" \
+  --session-arg "nft_holder_mode:u8='2'" \
+  --session-arg "minting_mode:u8='1'" \
+  --session-arg "metadata_mutability:u8='0'" \
+  --session-arg "burn_mode:u8='1'" \
+  --session-arg "nft_metadata_kind:u8='3'" \
+  --session-arg [JSON SCHEMA ESCAPED STRING]
+
+```
+replace [JSON SCHEMA ESCAPED STRING] with the escaped version of [above](#custom-metadata-example) example of a custom JSON SCHEMA:
+```
+"json_schema:string='{\"properties\":{\"nft_name\":{\"name\":\"nft_name\",\"description\":\"name_of_nft\",\"required\":true},\"nft_description\":{\"name\":\"nft_description\",\"description\":\"description_of_nft\",\"required\":true},\"nft_url\":{\"name\":\"nft_url\",\"description\":\"url_of_nft\",\"required\":true}}}'"
+```
+Use this external [tool](https://jsontostring.com/) to escape a custom JSON schema like [above](#custom-metadata-example) or the one from the Cep78 [readme](https://github.com/casper-ecosystem/cep-78-enhanced-nft#example-custom-validated-schema). Only escaped JSON metadata can be passed as a session argument when installing the contract or minting an NFT.
+
+## Mint an NFT
+Mint an NFT by calling the previously installed instance of the Cep-78 NFT standard.
+
+### Query your account
+```bash
+casper-client get-state-root-hash <node-address>
+```
+-> outputs a _state-root-hash_
+```bash
+casper-client get-account-state --node-address<NODE_ADDRESS> --state-root-hash <STATE_ROOT_HASH> --key <account-hash-YOUR_ACCOUNT_HASH>
+```
+_get-account-state_ returns a list of associated keys. One of these associated keys should be the _contract hash_ of the Cep-78 instance.
+
```

Comment:
This doesn't work for me; It asks me to use get-account-info, not get-account-state. Are we working with different cersions of casper client perhaps?